api_app/queryset.py

```diff
@@ -210,11 +210,17 @@ def visible_for_user(self, user: User = None) -> "PluginConfigQuerySet":
                 # we don't need to do anything.
                 return self.filter(Q(owner=user) | Q(owner__isnull=True))
             else:
-                return self.filter(
-                    (Q(for_organization=True) & Q(owner=membership.organization.owner))
-                    | Q(owner=user)
-                    | Q(owner__isnull=True)
-                )
+                if membership.is_admin:
+                    return self.filter(Q(for_organization=True) | Q(owner__isnull=True))
+                else:
+                    return self.filter(
+                        (
+                            Q(for_organization=True)
+                            & Q(owner=membership.organization.owner)
+                        )
+                        | Q(owner=user)
+                        | Q(owner__isnull=True)
+                    )
```

Comment:
Unfortunately, with this condition, if you are an `admin` you can see the configuration of **every** other organization user values. This is obviously not intended. 
1 - Can we add a test to verify or refute this claim?
2 - If the claim is valid, I would suggest something like
```python3
if membership.is_admin:

    Q(for_organization=True, owner__membership__organization=membership.organization) | 
    Q(owner=user) | Q(owner_isnull=True)
```
The test that you will make should fail before and pass after this change

Reply:
I was reasoning that in this case being admin/owner/user is indifferent to the level of visibility of the configs in the org...
I think the following code should be enough:
```python
# If you are member of an organization you should see the configs.
return self.filter(
    Q(for_organization=True, owner__membership__organization=membership.organization)
    | Q(owner=user)
    | Q(owner__isnull=True)
)
```